adcm_aio_client/objects/_cm.py

```diff
@@ -913,6 +968,13 @@ async def wait(
         poll_interval: int = 10,
         exit_condition: Callable[[Self], Awaitable[bool]] = default_exit_condition,
     ) -> Self:
+        """
+        Wait for `Job` to complete.
+        :param timeout: If the Job is not completed after the `timeout`, the `WaitTimeoutError` is raised
+        :param poll_interval: The interval at which the `Job`'s exit condition is checked
+        :param exit_condition: Callable of one argument: self. Calls on each iteration to check if the `Job` can be
+                               considered completed, regardless of an actual `Job`'s status.
```

Comment:
"completed" isn't a precise term here: users may wait for any condition they want, not only completeness

Reply:
rewrited